_docs/create-lightning-wallet.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to create a Lightning wallet
+tags: [setup]
+---
+
+Creating a Lightning wallet on BlueWallet is easy.
+
+You can either click on the "Add now" button or the "+" icon in the top right corner
+
+{% include phone-screenshot-image.html img="bluewallet-first-page.png" alt="BlueWallet's first page" %}
+
+That will take you to the "add wallet" screen.
+
+## Adding a new wallet
+
+In the "add wallet" screen, you can choose between a Bitcoin wallet and a Lightning wallet.
+
+{% include phone-screenshot-image.html img="add-lightning-wallet.png" alt="Add Lightning wallet" %}
+
+For Lightning, click on the Lightning button. (If you want to create a Bitcoin wallet, see [this guide](/docs/create-bitcoin-wallet).)
+
+You may also name your wallet, or leave the default name.
+
+<div class="side-note" markdown="1">
+
+### Advanced options
+
+If you have "Advanced mode" enabled in general settings,
+you'll see one more option for creating the wallet.
+
+{% include phone-screenshot-image.html img="add-lightning-wallet-with-advanced-options.png" alt="Add Lightning wallet with advanced options" %}
+
+#### Connecting to your own LNDHub
+
+Lightning functionality in BlueWallet is powered by [LNDHub](http://lndhub.io/), an open source service that we created for this purpose. We host our own instance of this service and BlueWallet connects to it by default. However, for more privacy you can host your own instance and connect to it.
+
+To do that, you can add the address of your node in the specified text field.
+
+#### Provide entropy
+
+Is the "Provide entropy" feature applicable to Lightning wallets???
+
+</div>
+
+When you're ready, you can click on "Create" to make your new wallet.
+
+### Importing a wallet
+
+??? is it possible to import a Lightning wallet?
```

Comment:
Is it possible to import a Lightning wallet?

Reply:
Yes, you can have lightning wallet backup, which looks like this `lndhub://fcdb207deeasdf8693ce:26425252asdff49e8c1`